lib/form/scripts.js

```diff
@@ -81,65 +91,35 @@ const rendered = (type) => {
       });
     });
 
-    const scripts = `
-var allScripts = ${JSON.stringify(scriptFunctions)};
-var allScriptInputs = ${JSON.stringify(inputs)};
-var allScriptSettings = ${JSON.stringify(allScriptSettings)};
-var allScriptIDs = ${JSON.stringify(ids)};
+    const formScript = `
+var allPlugins = ${JSON.stringify(allPlugins)};
```

Comment:
@Snugug since these are now global and will be used by multiple scripts (future and present) - can we please get a jsdocs breakdown of their objects?